src/uu/uniq/src/uniq.rs

```diff
@@ -265,6 +265,35 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         }
     };
 
+    let mut occrs = { 0..=9 }
```

Comment:
could you please add a comment explaining the why

Reply:
initially i used random letters because i didn't know how to name it, then i forgot to give it a proper name before committing